ports/espressif/boards/waveshare_esp32s2_pico/pins.c

```diff
@@ -33,9 +33,13 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_GP37), MP_ROM_PTR(&pin_GPIO37) },
     { MP_ROM_QSTR(MP_QSTR_GP38), MP_ROM_PTR(&pin_GPIO38) },
 
-    { MP_ROM_QSTR(MP_QSTR_IO39), MP_ROM_PTR(&pin_GPIO39) },
+    { MP_ROM_QSTR(MP_QSTR_GP39), MP_ROM_PTR(&pin_GPIO39) },
+
+    { MP_ROM_QSTR(MP_QSTR_40), MP_ROM_PTR(&pin_GPIO40) },
+    { MP_ROM_QSTR(MP_QSTR_41), MP_ROM_PTR(&pin_GPIO41) },
```

Comment:
Typo - missing "GP" on the pins.

I don't see any silkscreen labeling of SDA and SCL on the board. I would suggest taking these away, since they are not "default I2C", as far as I can tell.

```suggestion
    { MP_ROM_QSTR(MP_QSTR_GP40), MP_ROM_PTR(&pin_GPIO40) },
    { MP_ROM_QSTR(MP_QSTR_GP41), MP_ROM_PTR(&pin_GPIO41) },
```

Reply:
Sorry for the typo. It is indeed not default i2c, or written on the silkscreen as sda-scl.
Removed, as suggested.